test/test_nn.py

```diff
@@ -10342,12 +10342,58 @@ def test_linear_broadcasting(self):
         expected = m(inp.view(6, 5)).view(2, 3, 8)
         self.assertEqual(expected, m(inp))
 
+    @parametrize_test('device', ['cpu'] + (['cuda'] if TEST_CUDA else []))
+    @parametrize_test('bias', [
+        subtest(False, name='nobias'), subtest(True, name='bias')])
+    @parametrize_test('weight_layout', [
+        subtest(torch.strided, name='weightStrided'),
+        # TODO: NotImplementedError: Could not run 'aten::sparse_dim' with arguments from the 'CPU' backend
```

Comment:
nit: the PR below changes this, right?

Reply:
Right. The NotImplementedError is replaced with another RuntimeError but the TODO comment is only temporary, https://github.com/pytorch/pytorch/pull/86302 removes it.